decidim-core/app/packs/stylesheets/decidim/_header.scss

```diff
@@ -115,7 +115,7 @@ header {
       @apply fixed bottom-0 left-0 z-40 bg-white w-full px-4 py-3 flex justify-between text-secondary shadow-[0_-4px_6px_rgba(198,198,198,0.25)];
 
       &__trigger {
-        @apply flex flex-col items-center text-secondary cursor-pointer pl-4 border-l border-gray-3;
+        @apply flex flex-col items-center text-secondary cursor-pointer md:pl-4 md:border-l border-gray-3;
```

Comment:
```suggestion
        @apply flex flex-col items-center text-secondary cursor-pointer pl-2 md:pl-4 md:border-l border-gray-3;
```

Reply:
Suggesting add some left padding on mobile, to avoid to be stacked with the right part.

Reply:
![image](https://github.com/user-attachments/assets/df52ca9d-ccb7-46c5-901f-b3c82a908d6e)